src/analysis/signature_analysis.rs

```diff
@@ -10,7 +11,7 @@ use std::collections::BTreeMap;
 use std::io::{BufRead, BufReader};
 use std::fs::File;
 use glob::glob;
-use crc::crc16;
+//use crc::crc16;
```

Comment:
i'll have to fix this

Reply:
Yeah I turned it off temporarily for internal sigs.